torch_frame/data/dataset.py

```diff
@@ -316,7 +335,15 @@ def materialize(self, device: Optional[torch.device] = None,
 
         # 1. Fill column statistics:
         for col, stype in self.col_to_stype.items():
-            self._col_stats[col] = compute_col_stats(self.df[col], stype)
+            if stype == torch_frame.multi_categorical:
+                ser = self.df[col].apply(
+                    lambda x: [cat.strip() for cat in x.split(self.sep)]
+                    if x is not None and x != '' else [])
```

Comment:
this needs to happen in tensor mapper, right?

Reply:
Yeah, but when we calculate column stats, where we calculate the necessary arguments for Tensor Mapper, we need to split the columns by separator so we know what is the occurrence of each category.

Reply:
I get this. But I would have introduced a new stat type that does this splitting internally, and requires `sep` as an argument.